src/Akka.Streams.Kafka/Messages/CommittableMessage.cs

```diff
@@ -31,17 +32,17 @@ public CommittableMessage(ConsumerRecord<K, V> record, CommitableOffset commitab
     /// </summary>
     public class CommitableOffset
     {
-        private readonly Func<CommittedOffsets> _task;
+        private readonly Func<List<TopicPartitionOffset>> _task;
```

Comment:
Why the change?

Reply:
Mostly because this CommitableOffset class is instantiated from consumer.Commit response like this:

``` csharp
var commitableOffset = new CommitableOffset(
    () => consumer.Commit(),
    new PartitionOffset("groupId", message.Topic, message.Partition, message.Offset));
```

And now Commit returns `List<TopicPartitionOffset>` instead of `CommittedOffsets`.

Reply:
Ah got it - understood.